packages/lesswrong/lib/sql/AddFieldQuery.ts

```diff
@@ -0,0 +1,25 @@
+import Query from "./Query";
+import Table from "./Table";
+
+/**
+ * Builds a Postgres query to add a new field to a Postgres table.
+ * This assumes that the field already exists in the schema but
+ * has not been propogated to the database yet and, as such, is
+ * mainly meant for use in migrations.
```

Comment:
```suggestion
 * meant for use in migrations.
```

Do you have another use case in mind? If not, don't hedge.